pnpm-lock.yaml

```diff
@@ -14,17 +14,17 @@ patchedDependencies:
   '@nuxtjs/i18n@9.1.0':
     hash: f7ooybjhk3gbrcmx6vkut3a44i
     path: patches/@nuxtjs__i18n@9.1.0.patch
-  nuxt-security:
-    hash: ioplwzeg3uw7dx44sezdh4wsmi
-    path: patches/nuxt-security.patch
-  pinceau:
-    hash: d6ha36xrn7oh52pyhfdxwv3tsq
-    path: patches/pinceau.patch
```

Comment:
It looks like these necessary lines were somehow deleted in this change. I think those are not expected to be removed when just adding new packages.

Could you try reverting these changes and adding new packages + run `pnpm i` again?

Reply:
should be fixed now